tests/framework/unit_tests/utils/testRandomUtils.py

```diff
@@ -214,7 +214,7 @@ def checkType(comment,value,expected,updateResults=True):
 ### randomIntegers(), sampling integers in a range
 randomUtils.randomSeed(42,engine=None)
 randomUtils.randomSeed(42,engine=eng)
-right = [14,18,20,12,18]
+right = [14,18,20,12,17]
```

Comment:
I think the changes in this file are a mistake.

Reply:
fixed